test/BUILD.gn

```diff
@@ -260,6 +260,7 @@ test("brave_unit_tests") {
       "//brave/app/brave_command_line_helper_unittest.cc",
       "//brave/browser/resources/settings/brandcode_config_fetcher_unittest.cc",
       "//brave/browser/resources/settings/reset_report_uploader_unittest.cc",
+      "//brave/browser/search/ntp_utils_unittest.cc",
```

Comment:
We recommend to make separate test target in their home instead of adding here directly for new test cases.

Reply:
fixed, thanks